examples/src/pages/SimplePointcloud.tsx

```diff
@@ -100,10 +100,22 @@ export function SimplePointcloud() {
         throw new Error(
           'Need to provide either project & model OR modelUrl as query parameters'
         );
-      }
+      }      
       scene.add(pointCloudNode);
       revealManager.on('loadingStateChanged', setLoadingState);
 
+      const classesGui = gui.addFolder('Class filters');
+      const enabledClasses: { [clazz: number]: boolean } = {};
+      console.log(pointCloudNode.getClasses());
+      for (const clazz of pointCloudNode.getClasses()) {
```

Comment:
@larsmoa I know it's internal but still... do we really need to access the node instead of using public API here? Enum is exported. You can get its values and keys. This is the code I'm currently using in the console to get a clean label/value map.

```js
 Object.keys(WellKnownPointClassTypes)
      .filter(key => Number.isNaN(Number(key))) // filter out enum numbers that stored as keys
      .reduce((acc, key) => {
        acc[key] = WellKnownPointClassTypes[key];
        return acc;
      }, {})
```

Don't really insist on changes here, but just in case posted it here.

Reply:
Yes, the enum only has a few 'pre-defined' codes, the user can define custom values so we can't enumerate all of them.